src/main/resources/interface-specification.yml

```diff
@@ -628,17 +692,18 @@ components:
           format: uuid
         name:
           type: string
-          pattern: /^[a-zA-Z]/s+$/
-        version:
+        category:
           type: string
-          pattern: ^[0-9]{1,4}$
-        state:
-          $ref: '#/components/schemas/EServiceDescriptorState'
       required:
         - id
         - name
-        - version
-        - state
+        - category
+    Origin:
+      description: Attribute origin admitted
+      type: string
+      enum:
+        - IPA
+        - SDG
```

Comment:
this will need to be maintained manually

Reply:
Exactly. Did I miss some decision about it? Why don't expose a mere string?

Reply:
If I remember, it was intended also as "documentation" of the existing origins, but I'm not sure if it's worth.

@beetlecrunch what do you think?
(given that we are still in the test phase, we should be able to change this before October)